src/connectors/hits/connectHits.js

```diff
@@ -88,6 +90,11 @@ export default function connectHits(renderFn, unmountFn) {
 
         results.hits = transformItems(results.hits);
 
+        // make sure the escaped tag stays, even after mapping over the hits
+        // this prevents the hits from being double-escaped if there are multiple
```

Comment:
```suggestion
        // This prevents the hits from being double-escaped if there are multiple
```

Reply:
Can we add a lint rule instead

Reply:
It's more that the sentence was not correct at first because it needed a break in between (a fullstop here), so then we should capitalize the next sentence.